packages/core/package.json

```
@@ -3541,6 +3625,155 @@
+                "title": "%AWS.command.docdb.rebootInstance%",
+                "icon": "$(debug-restart)",
+                "category": "%AWS.title%",
+                "enablement": "(isCloud9 || !aws.isWebExtHost) && viewItem == awsDocDB-instance-available",
+                "cloud9": {
+                    "cn": {
+                        "category": "%AWS.title.cn%"
+                    }
+                }
+            },
+            {
+                "command": "aws.docdb.renameCluster",
+                "title": "%AWS.generic.promptRename%",
+                "category": "%AWS.title%",
+                "enablement": "(isCloud9 || !aws.isWebExtHost) && viewItem =~ /^awsDocDB-cluster.*-running/",
+                "cloud9": {
+                    "cn": {
+                        "category": "%AWS.title.cn%"
+                    }
+                }
+            },
+            {
+                "command": "aws.docdb.startCluster",
+                "title": "%AWS.command.docdb.startCluster%",
+                "icon": "$(debug-start)",
+                "category": "%AWS.title%",
+                "enablement": "!aws.isWebExtHost && viewItem =~ /^awsDocDB-cluster.*-stopped/"
+            },
+            {
+                "command": "aws.docdb.stopCluster",
+                "title": "%AWS.command.docdb.stopCluster%",
+                "icon": "$(debug-pause)",
+                "category": "%AWS.title%",
+                "enablement": "!aws.isWebExtHost && viewItem =~ /^awsDocDB-cluster.*-running/"
+            },
+            {
+                "command": "aws.docdb.deleteCluster",
+                "title": "%AWS.generic.promptDelete%",
+                "category": "%AWS.title%",
+                "enablement": "(isCloud9 || !aws.isWebExtHost) && viewItem =~ /^awsDocDB-cluster.*-running/",
+                "cloud9": {
+                    "cn": {
+                        "category": "%AWS.title.cn%"
+                    }
+                }
+            },
+            {
+                "command": "aws.docdb.listTags",
+                "title": "%AWS.command.docdb.tags%",
+                "category": "%AWS.title%",
+                "enablement": "(isCloud9 || !aws.isWebExtHost) && viewItem =~ /^awsDocDB/",
+                "cloud9": {
+                    "cn": {
+                        "category": "%AWS.title.cn%"
+                    }
+                }
+            },
+            {
+                "command": "aws.docdb.openBrowser",
+                "title": "%AWS.command.docdb.open%",
+                "category": "%AWS.title%",
+                "enablement": "(isCloud9 || !aws.isWebExtHost) && viewItem =~ /^awsDocDB/",
+                "cloud9": {
+                    "cn": {
+                        "category": "%AWS.title.cn%"
+                    }
+                }
+            },
+            {
+                "command": "aws.docdb.openHelp",
+                "title": "%AWS.command.docdb.openHelp%",
```

let's use the same name that we do for others. that will help in the future when we generalize the "view help/docs" action.

```suggestion
                "command": "aws.docdb.viewDocs",
                "title": "%AWS.generic.viewDocs%",
```